### DISCUSSION TITLE
When does DOM reflow occur?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#213] [Q&A] [answered]
In short.

**Comment by fcasamento** [comment, marked as answer]
@montasim I believe it happens when: Reflow is the name of the web browser process for re-calculating the positions and geometries of elements in the document, for the purpose of re-rendering part or all of the document.

**Reply by montasim** [reply]
Hey, @fcasamento do you have any idea about this? https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions/301
